testscommon/state/testingTriePruningStorer.go

```diff
@@ -0,0 +1,64 @@
+package state
+
+import (
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/storage/memorydb"
+	storageMock "github.com/ElrondNetwork/elrond-go/storage/mock"
+	"github.com/ElrondNetwork/elrond-go/storage/pruning"
+	"github.com/ElrondNetwork/elrond-go/storage/storageUnit"
+	"github.com/ElrondNetwork/elrond-go/testscommon"
+)
+
+// CreateTriePruningStorer creates a new trie pruning storer that is used for testing
+func CreateTriePruningStorer(coordinator sharding.Coordinator, notifier pruning.EpochStartNotifier) (storage.Storer, map[string]storage.Persister, error) {
```

Comment:
CreateTestingTriePruningStorer to emphasize the type of trie pruning storer?
Also, can we move this into the `testscommon` package?

Reply:
Renamed. Connot move due to cyclic imports.

Reply:
it is already in the testscommon package :man_facepalming:

Reply:
It is in state package from the testcommon package.